vscode-lean4/src/utils/fsExistHelper.ts

```diff
@@ -0,0 +1,29 @@
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
+import { open, close, PathLike } from 'fs';
+
+
+export function fsExistHelper(pathFile: PathLike): boolean {
```

Comment:
What is the purpose of this helper? It is not asynchronous, so it is just a reimplementation of `fs.existsSync`, no? In the previous PR version there was an `async` variant.

Reply:
I will change this into an async function :)